packages/e2e-tests/src/selectors.ts

```diff
@@ -256,6 +272,32 @@ export class Channel {
     )
   }
 
+  async getUserMessagesWrapper(username: string) {
+    return await this.driver.wait(
+      until.elementsLocated(By.xpath(`//*[contains(@data-testid, "userMessagesWrapper-${username}")]`))
+    )
+  }
+
+  async waitForLabel(username: string, label: string) {
+    console.log(`Waiting for user's "${username}" label "${label}" label`)
+    await this.driver.wait(async () => {
+      const labels = await this.driver.findElements(By.xpath(`//*[contains(@data-testid, "userLabel-${username}")]`))
+      const properLabels = labels.filter(async labelElement => {
+        const labelText = await labelElement.getText()
+        return labelText === label
+      })
+      return properLabels.length > 0
+    })
+  }
+
+  async waitForLabelsNotPresent(username: string) {
```

Comment:
Are these two functions being used anywhere?

Reply:
waitForLabel* - yes, in multipleClients.test.ts
getUserMessagesWrapper - no, that one I can delete.

Reply:
Ok, if it's not being used, I'd suggest removing it unless it's going to be used soon in a subsequent PR.